demos/browser/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "./build",
+    "rootDir": "../.."
```

Comment:
Also due to this, does it need to have `composite` set to true in the root `tsconfig.json`? I checked TS compiler is giving a red alert for this. Demo works though without adding that composite true with this config.